src/CodeStyle/CSharp/Analyzers/PublicAPI.Shipped.txt

```diff
@@ -1,56 +0,0 @@
-Microsoft.CodeAnalysis.CSharp.Formatting.BinaryOperatorSpacingOptions
```

Comment:
Note the public API files for CodeStyle layer are not really an API contract. There were added to track how many conflicting public types we have in CodeStyle and Workspaces layer. The goal is to get these public API files to be empty, or at least avoid any conflicting types being here.